### ISSUE TITLE
lambdify of Transpose is wrong

### ISSUE THREAD

**Issue by asmeurer** [sympy/sympy#11044]
Reported from https://stackoverflow.com/questions/36730931/converting-expression-involving-tranpose-of-vector-to-numerical-function-with-la

```
In [19]: x = sympy.MatrixSymbol('x',3,1)

In [20]: f = sympy.lambdify(x, x.T*x)
  File "<string>", line 1
    lambda x: ((x').dot(x))
                          ^
SyntaxError: EOL while scanning string literal
```

On a related note, the `str` printer for Transpose should print valid Python.

**Comment by Nitin216** [comment]
Without even multiplication of ```x.T`` with ```x```. It is still giving same error and its due to ```eval(lstr,namespace)``` statement in lambdify.py.I found that but can't see how to remove that.Can someone push me in right direction.

**Comment by asmeurer** [comment]
One need to change the `str` printer for Transpose (in sympy/printing/str.py) so that it prints `x.T` instead of `x'`. 

I think lambdify still won't work after that, but it won't give a SyntaxError.

**Comment by Nitin216** [comment]
@asmeurer Thank you. Just gonna do it now.

**Comment by sdewaele** [comment]
I encountered this issue with sympy 1.0. It would be great it could be solved. It seems to be addressed properly by PR #11046.

**Comment by gxyd** [comment]
Seems like @asmeurer forgot and opened a duplicate issue for 'str' printing here https://github.com/sympy/sympy/issues/11559 which has been fixed.

As for the `lambdify` error, it has been fixed by PR https://github.com/sympy/sympy/pull/11566 . Only task remains is to add a test case as @asmeurer suggested here https://github.com/sympy/sympy/pull/11046#issuecomment-299757406. I'm closing this one, I'll open another issue explaining what needs to be done.

**Comment by gxyd** [comment]
@sdewaele this has been fixed in sympy version >= 1.1, see https://github.com/sympy/sympy/pull/11566

**Comment by gxyd** [comment]
Opened https://github.com/sympy/sympy/issues/13881 for adding the test case for lambdify
